manager/integration/tests/test_scheduling.py

```diff
@@ -7,11 +7,9 @@
     wait_for_volume_detached, wait_for_volume_degraded, \
     wait_for_volume_healthy, wait_scheduling_failure, \
     write_volume_random_data, wait_for_rebuild_complete
-
+from common import SETTING_REPLICA_SOFT_ANTI_AFFINITY
```

Comment:
This should be in the previous commit since the constant was moved then.

This commit should deal with SETTING_ZONE_SOFT_ANTI_AFFINITY only.

Also, now it shows that the naming of two constants is inconsistent. We call them either:
`SETTING_REPLICA_NODE_SOFT_ANTI_AFFINITY` and `SETTING_REPLICA_ZONE_SOFT_ANTI_AFFINITY`
Or:
`SETTING_NODE_SOFT_ANTI_AFFINITY` and `SETTING_ZONE_SOFT_ANTI_AFFINITY`

But `SETTING_REPLICA_SOFT_ANTI_AFFINITY` and `SETTING_ZONE_SOFT_ANTI_AFFINITY` doesn't work.

Reply:
fixed